src/requirements/data/exams/ExamCredit.ts

```diff
@@ -1,316 +1,443 @@
+//         },
+//         {
+//             // TODO
+//             subject: "Spanish Language",
+//             credits: [3],
+//             collegesApplied: [],
+//             majorExcluded: [],
+//             mininmumScore: 4,
+//             courseEquivalents: [],
+//             requirementEquivalents: [],
+//         },
+//         {
+//             // TODO
+//             subject: "Spanish Literature",
+//             credits: [3],
+//             collegesApplied: [],
+//             majorExcluded: [],
+//             mininmumScore: 4,
+//             courseEquivalents: [],
+//             requirementEquivalents: [],
+//         },
+//         {
+//             // TODO
+//             subject: "Statistics",
+//             credits: [3],
+//             collegesApplied: [],
+//             majorExcluded: [],
+//             mininmumScore: 4,
+//             courseEquivalents: [[ 'AEM 2100', 'BTRY 3010', 'BTRY 6010',
+//             'ILRST 2100', 'ILRST 6100', 'MATH 1710', 'PAM 2100',
+//             'PAM 2101', 'PSYCH 2500', 'SOC 3010', 'STSCI 2100', 'STSCI 2150', 'STSCI 2200']],
+//             requirementEquivalents: [],
+//         }
+//         */,
+//   ],
+
+//   IB: [
+//     {
+//       subject: 'Chemical And Physical Systems',
+//       credits: [
+//         {
+//           operator: 'or',
+//           collegesApplied: ['EN', 'AS', 'AR'],
+//           majorExcluded: ['CHEM'],
+//           mininmumScore: 5,
+//           courseEquivalents: ['CHEM 2070', 'CHEM 2090'],
+//           requirementEquivalents: [],
+//           credits: 4,
+//         },
+//       ],
+//     },
+//     /*
+//         {
+//             subject: "Chemistry",
+//             credits: [4],
+//             collegesApplied: ["EN", "AS"],
+//             majorExcluded: [],
+//             mininmumScore: 6,
+//             courseEquivalents: [['CHEM 2070', 'CHEM 2090']],
+//             requirementEquivalents: []
+//         },
+//         {
+//             subject: "Computer Science",
+//             credits: [4],
+//             collegesApplied: ["EN", "AS"],
+//             majorExcluded: [],
+//             mininmumScore: 6,
+//             courseEquivalents: [['CS 1110']],
+//             requirementEquivalents: []
+//         },
+//         {
+//             subject: "Economics",
+//             credits: [6],
+//             collegesApplied: ["EN", "AS"],
+//             majorExcluded: [],
+//             mininmumScore: 6,
+//             courseEquivalents: [['CS 1110']],
+//             requirementEquivalents: []
+//             // varies
+
+//         },
+//         {
+//             subject: "English",
+//             credits: [6],
+//             collegesApplied: ["EN", "AS"],
+//             majorExcluded: [],
+//             mininmumScore: 6,
+//             courseEquivalents: [['CS 1110']],
+//             requirementEquivalents: []
+//             // varies
+//         },
+//         */
+//   ],
+// };
```

Comment:
what's happening with these commented code

Reply:
It's the old code (with the old schema). I was planning on adding the correct data in another PR.

Reply:
I'm cool leaving it commented out like this as long as Sam is